Person/persons.md

```diff
@@ -0,0 +1,386 @@
+*familyName* | `String` | family name/surname of the person
+*familyName2* | `String` | second family name/surname of the person
+*preferred* | `Boolean` | preferred/not preferred name
+*prefix* | `String` | prefix for the name
+*familyNamePrefix* | `String` | prefix if any for the family name 
+*familyNameSuffix* | `String` | Suffix if any for the family name
+*degree* | `String` | degree attained by the person
+
+```console
+POST person/:target_person_uuid/name
+{ 
+    "givenName": "Mohit",
+    "familyName": "Kumar",
+    "preferred": true,
+    "prefix": "Mr."
+}
+```
+
+### Update person name subresource
+
+* To update a person name with given uuid value for a specific person resource you need to specify below properties in your request body.
+If user not logged in to perform this action, a `401 Unauthorized` status returned.
+
+#### Attributes
+
+Parameter | Type | Description
+--- | --- | ---
+*givenname* | `String` | name of the person
+*middlename* | `String` | middle name of the person
+*familyName* | `String` | family name/surname of the person
+*familyName2* | `String` | second family name/surname of the person
+*preferred* | `Boolean` | preferred/not preferred name
+*prefix* | `String` | prefix for the name
+*familyNamePrefix* | `String` | prefix if any for the family name 
+*familyNameSuffix* | `String` | Suffix if any for the family name
+*degree* | `String` | degree attained by the person
+
+```console
+POST person/:target_person_uuid/name
+{ 
+    "givenName": "Mohit",
+    "familyName": "Verma",
+    "preferred": true,
+    "prefix": "Dr."
+}
+```
+
+### Delete a person name sub resource
+
+* Delete or retire a target name sub resource by its UUID.Returns a `404 Not Found` status if attribute not exists. 
+If user not logged in to perform this action, a `401 Unauthorized` status returned.
+
+ ```console
+DELETE /person/:target_person_uuid/person/:target_name_uuid
+ ```
+
+### List person address subresource
+
+* List all the person address subresource corresponding to a `target_person_uuid`. Returns a `404 Not Found` status if person address not exists. If user not logged in to perform this action, a `401 unauthorized` status returned.
+
+```console
+GET /person/:target_person_uuid/address
+```
+
+* List all the person address subresource by it's `target_address_uuid` and corresponding to a `target_person_uuid`. Returns a `404 Not Found` status if person address not exists. If user not logged in to perform this action, a `401 unauthorized` status returned.
+
+```console
+GET /person/:target_person_uuid/address/:target_address_uuid
+```
+
+### Create person address subresource
+
+* To create a person address sub resource for a specific person resource you need to specify below properties in your request body.
+If user not logged in to perform this action, a `401 Unauthorized` status returned.
+
+#### Attributes
+
+Parameter | Type | Description
+--- | --- | ---
+*preferred* | `Boolean` | preferred/not preferred address
```

Comment:
`true` if this is the person's preferred address. If a person has multiple addresses, only one can be preferred.